README.md

```diff
@@ -107,17 +107,23 @@ Depending on how ORT was installed, it can be run in the following ways:
 
 * If the Docker image was built, use
 
-      docker run ort --help
+  ```bash
```

Comment:
We sometimes also used the more generic "shell", which is sufficient here. As I don't think we have any Bash-specific code blocks, would it make sense to align all code blocks on "shell"?

Reply:
I have updated the commit and added another one to align on `shell`.